packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -161,6 +161,7 @@ class DateTimePicker extends React.Component {
   }
 
   handleOnDateBlur(event) {
+    debugger;
```

Comment:
We should remove the `debugger;` statement here

Reply:
https://github.com/cerner/terra-framework/pull/573/commits/e5022b9b87707ab87e848ee59e1b73eca9442c18